### ISSUE TITLE
Output garbled under tox 2.x

### ISSUE THREAD

**Issue by ulope** [Frozenball/pytest-sugar#60]
When running under tox >= 2.0 the output shows raw escape sequences.
It might well be a tox bug but OTOH pytest's own colored output works just fine.

Example output:
```
py34 inst-nodeps: xxxx.zip
py34 installed: coverage==4.0a5,xxxx==1.0.0a0,flake8==2.4.1,future==0.14.3,mccabe==0.3,pep8==1.5.7,py==1.4.27,pyflakes==0.8.1,pytest==2.7.1,pytest-mock==0.5.0,pytest-sugar==0.4.0,termcolor==1.1.0,wheel==0.24.0
py34 runtests: PYTHONHASHSEED='3528495582'
py34 runtests: commands[0] | coverage run --source xxxx --branch -m py.test --junitxml=junit-py34.xml
Test session starts (platform: darwin, Python 3.4.3, pytest 2.7.1, pytest-sugar 0.4.0)
rootdir: /Users/ulo/Dropbox/devel/xxxx, inifile:
plugins: mock, sugar

\r 
\x1b[36mtests/\x1b[0mtest_config.py\x1b[0m \x1b[32m\u2713\x1b[0m                                                                                                                                                                         25% \x1b[40m\x1b[32m\u2588\u2588\u258c       \x1b[0m\r 
\x1b[36mtests/\x1b[0mtest_config.py\x1b[0m \x1b[32m\u2713\x1b[0m\x1b[32m\u2713\x1b[0m                                                                                                                                                                        50% \x1b[40m\x1b[32m\u2588\u2588\u2588\u2588\u2588     \x1b[0m\r 
\x1b[36mtests/\x1b[0mtest_config.py\x1b[0m \x1b[32m\u2713\x1b[0m\x1b[32m\u2713\x1b[0m\x1b[32m\u2713\x1b[0m                                                                                                                                                                       75% \x1b[40m\x1b[32m\u2588\u2588\u2588\u2588\u2588\u2588\u2588\u258c  \x1b[0m\r 
\x1b[36mtests/\x1b[0mtest_config.py\x1b[0m \x1b[32m\u2713\x1b[0m\x1b[32m\u2713\x1b[0m\x1b[32m\u2713\x1b[0m\x1b[32m\u2713\x1b[0m                                                                                                                                                                     100% \x1b[40m\x1b[32m\u2588\u2588\u2588\u2588\u2588\u2588\u2588\u2588\u2588\u2588\x1b[0m
```
(I added a linebreak after each `\r` to increase readability)

**Comment by RonnyPfannschmidt** [comment]
this is probably related to tox 2.0 environment isolation

**Comment by ulope** [comment]
Looks like this has been fixed in tox >= 2.0.2
